### PR TITLE
SecurityOfficerOMAS

### PR DESCRIPTION
**Pull request by danielaotelea**
1. process the "new relationship" -semantic assigment between a glossary term and a schema element
2. publish on its output topic the event that contains the security tag, schema element, business term
3. rest endpoints: to save/update the classification for the schema element and fetching the security tab based on the schema element id

Signed-off-by: Daniela Otelea <daniela-valentina.otelea@ing.com>

### CODE REVIEW

File: open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java [hunk 1]
```diff
@@ -699,9 +700,34 @@ public Connection getSecuritySyncServerConnection(String              serverName
...
+     * @param serverName  name of the real repository server
+     * @param url  url for the Security Officer Server
+     * @param configurationProperties name value pairs for the connection
+     * @return Connection object
+     */
+    public Connection getSecurityOfficerServerConnection(String              serverName,
+                                                         String              url,
+                                                         Map<String, Object> configurationProperties)
+    {
+        final String endpointGUID          = "P66yJ8gI-gJaZ-VOHw-b5fu-bfdilclSvfm1";
```
Comment (planetf1) [inline open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java]: GUIds are typically hex --this looks quite different to that which we usually use. Whilst it is interpreted as a string, I think we should stick with a standard approach?

File: open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java [hunk 2]
```diff
@@ -699,9 +700,34 @@ public Connection getSecuritySyncServerConnection(String              serverName
         String connectionName = "SecuritySyncServer.Connection." + serverName;
         final String connectionGUID        = "GPKHhpfW-xXjN-QbFR-T50l-XMnY68SS2Bip";
```
Comment (planetf1) [inline open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java]: non hex guid?

Comment (planetf1) [reply open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java]: Note there were others in the commit  too

File: open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java (hunk 1 above)
Comment (danielaotelea) [reply open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java]: updated in the last commit.

File: open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/security-officer-connectors/security-officer-tag-connector/src/main/java/org/odpi/openmetadata/openconnector/governancedarmonconnectors/securityofficerconnectors/securitytagconnector/SecurityTagConnectorProvider.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.openconnector.governancedarmonconnectors.securityofficerconnectors.securitytagconnector;
+
+import org.odpi.openmetadata.frameworks.connectors.ConnectorProviderBase;
+import org.odpi.openmetadata.frameworks.connectors.properties.beans.ConnectorType;
+
+public class SecurityTagConnectorProvider extends ConnectorProviderBase {
+
+    static final String connectorTypeGUID = "jghsfXyV-bEhX-G4cO-gNl2-qqEKqbqCUbWS";
```
Comment (planetf1) [inline open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/security-officer-connectors/security-officer-tag-connector/src/main/java/org/odpi/openmetadata/openconnector/governancedarmonconnectors/securityofficerconnectors/securitytagconnector/SecurityTagConnectorProvider.java]: non hex GUID

File: open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java (hunk 1 above)
Comment (planetf1) [reply open-metadata-implementation/adapters/open-connectors/connector-configuration-factory/src/main/java/org/odpi/openmetadata/adapters/repositoryservices/ConnectorConfigurationFactory.java]: Thanks -- I saw most have been changed - just noticed one left :-)

**Comment by planetf1** [comment]
still one rogue guid in open-metadata-implementation/adapters/open-connectors/governance-daemon-connectors/security-officer-connectors/security-officer-tag-connector/src/main/java/org/odpi/openmetadata/openconnector/governancedarmonconnectors/securityofficerconnectors/securitytagconnector/SecurityTagConnectorProvider.java that I can see. Hope that helps.

**Comment by danielaotelea** [comment]
The GUID from SecurityTagConnectorProvider has been changed.

**Pull request merged**